### ISSUE TITLE
visualization of inference/convergence diagnostics

### ISSUE THREAD

**Issue by dustinvtran** [blei-lab/edward#353]
for example, this includes:

+ trace plots for MCMC
+ loss function values by iteration for variational inference
+ parameter estimates or a predicted quantity by iteration

tensorboard is perfect for tracking these. all should live in tensorboard.

references
+ [stan's bayesplot package](https://github.com/stan-dev/bayesplot)
+ [pymc3 plots](https://pymc-devs.github.io/pymc3/api.html#module-pymc3.plots)
+ [seaborn examples](http://seaborn.pydata.org/examples/)
+ [bokeh gallery](http://bokeh.pydata.org/en/latest/docs/gallery.html)
+ [scikit-learn examples](http://scikit-learn.org/stable/auto_examples/)

**Comment by dustinvtran** [comment]
#654 gets us 90% here. The last step is MCMC features. Namely, it would be nice to display essential MCMC plots such as a trace plot over the first X many dimensions of a parameter (and as usual, `tf.summary.histogram` to see all dimensions at once).

**Comment by dustinvtran** [comment]
Closing this and adding a new issue for MCMC + TensorBoard.
